python/conversion/format.py

```diff
@@ -11,13 +13,20 @@
     import conversion.prophesee
 except ImportError:
     print("Conversion from .raw is not possible. If you want to extract .raw files, please install Metavision 2.2")
-    metavision_found = False
-
+    metavision_found_2_2 = False
```

Comment:
we should rename `metavision_found_2_2` to something more expressive. My proposal: `raw_conversion_possible`